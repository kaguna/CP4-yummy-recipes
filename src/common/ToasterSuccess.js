import React from 'react';
import { ToastContainer } from 'react-toastify';

const Toaster = () => (
  <ToastContainer
    position="top-right"
    type="default"
    autoClose={5000}
    hideProgressBar={true}
    closeOnClick
  />
);
export default Toaster;
