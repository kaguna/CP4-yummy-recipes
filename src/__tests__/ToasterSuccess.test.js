import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import ToasterSuccess from '../common/ToasterSuccess';

describe('Toast success component', () => {
  const wrapper = shallow(<ToasterSuccess />);

  it('renders properly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
