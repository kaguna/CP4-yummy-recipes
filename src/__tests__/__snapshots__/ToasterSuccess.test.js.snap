// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast success component renders properly 1`] = `
<ToastContainer
  autoClose={5000}
  bodyClassName=""
  className={null}
  closeButton={
    <DefaultCloseButton
      ariaLabel="close"
    />
  }
  closeOnClick={true}
  hideProgressBar={true}
  newestOnTop={false}
  pauseOnHover={true}
  position="top-right"
  progressClassName=""
  style={null}
  toastClassName=""
  transition={[Function]}
  type="default"
/>
`;
